### PR TITLE
Update rnaseq.md

### PR DESCRIPTION
**Pull request by cebriggs7135**

### CODE REVIEW

File: context/app/markdown/docs/assay/rnaseq.md [hunk 1]
```diff
@@ -7,48 +7,51 @@
...
-A visual summary is provided below. [https://rpubs.com/ewilkinson_KRISP/479615](https://rpubs.com/ewilkinson_KRISP/479615)
-**![](https://lh5.googleusercontent.com/lucozpAWSmdTG4KDF7I9Wfua5rvVxX7WlSCdHnaOAS-oaS92o-96hN9OdGyzerGUXUaUIuim4l6JKqeuvAPLUFib_seMquWlu0aL-U4K_WSSIPdhdVUKD4aMXBysAMXoGcGbr4ev)**
+A visual summary is provided below.
+![](https://lh5.googleusercontent.com/lucozpAWSmdTG4KDF7I9Wfua5rvVxX7WlSCdHnaOAS-oaS92o-96hN9OdGyzerGUXUaUIuim4l6JKqeuvAPLUFib_seMquWlu0aL-U4K_WSSIPdhdVUKD4aMXBysAMXoGcGbr4ev)
+
+Figure 2: A basic representation of the major steps and considerations in the sequencing of RNA.
+[(Kukurba et al, 2015)](https://doi.org/10.1101/pdb.top084970)
+
 ### *10x Genomics single cell RNASeq* 
+Utilizes a droplet-based emulsion PCR method to encapsulate individual cells with enzyme-containing beads in oil droplets, tagging single cell transcriptomes with UMIs and unique cell barcodes via reverse transcription to cDNA. Tagged transcriptomes are subsequently amplified, sheared to appropriate fragment size, repaired, and ligated with sequencing adapters (illumina) and user-chosen sample index oligos
```
Comment (cebriggs7135) [inline context/app/markdown/docs/assay/rnaseq.md:33]:
```suggestion
Utilizes a droplet-based emulsion PCR method to encapsulate individual cells with enzyme-containing beads in oil droplets, tagging single cell transcriptomes with UMIs and unique cell barcodes via reverse transcription to cDNA. Tagged transcriptomes are subsequently amplified, sheared to appropriate fragment size, repaired, and ligated with sequencing adapters (illumina) and user-chosen sample index oligos.
- For questions, contact: [Maigan Brusko](mailto:maigan@ufl.edu)
```

File: context/app/markdown/docs/assay/rnaseq.md [hunk 2]
```diff
@@ -7,48 +7,51 @@
...
+*Figure 3: Single cell workflow [(Image from Roy J. Carver Biotechnology Center)](https://biotech.illinois.edu/htdna/applications)*.
 
   
 ### *SNARE-Seq*
-
-[https://www.nature.com/articles/s41587-019-0290-0](https://www.nature.com/articles/s41587-019-0290-0)
-
 Accessible chromatin in permeabilized nuclei is captured by Tn5 transposase, before droplet generation (DropSeq). Without heating or detergent treatment, binding of transposase to its DNA substrate after transposition maintains contiguity of the original DNA strands, allowing for the co-packaging of accessible genomic sites and mRNA from individual nuclei in the same droplets. A splint oligonucleotide with sequence complementary to the adaptor sequence inserted by transposition (5′ end) and the poly(A) bases (3′ end) allows capture by oligo(dT)-bearing barcoded beads. After encapsulation of nuclei, mRNAs and fragmented chromatin can be released by heating the droplets, allowing access to splint oligonucleotides and adaptor-coated beads with a shared cellular barcode for library construction
```
Comment (cebriggs7135) [inline context/app/markdown/docs/assay/rnaseq.md:42]:
```suggestion
For questions, contact: [Blue Lake](mailto:b1lake@eng.ucsd.edu)
```

File: context/app/markdown/docs/assay/rnaseq.md [hunk 3]
```diff
@@ -7,48 +7,51 @@
...
...PCR plate. (v) Amplicons are pooled and subjected to massively parallel sequencing, resulting in 3′-tag digital gene expression profiles, with each read associated with two barcodes corresponding to the first and second rounds of cellular indexing. In a variant of the method described below, we introduce a third round of cellular indexing during Tn5 transposition of double-stranded cDNA. Most cells pass through a unique combination of wells, resulting in a unique combination of barcodes for each cell that tags its transcripts. The rate of two or more cells receiving the same combination of barcodes can be tuned by adjusting how many cells are distributed to the second set of wells. Increasing the number of barcodes used during each round of indexing boosts the number of cells that can be profiled while reducing the effective cost per cell. Additional levels of indexing can potentially offer even greater complexity and lower costs. Multiple samples (e.g., from different cell populations, tissues, individuals, time points, perturbations, or replicates) can be concurrently processed in one experiment, using different subsets of wells for each sample during the first round of indexing.
```
Comment (cebriggs7135) [inline context/app/markdown/docs/assay/rnaseq.md:49]:
```suggestion
For questions, contact: [Dana L Jackson](mailto:danaj77@uw.edu)
```

File: context/app/markdown/docs/assay/rnaseq.md [hunk 4]
```diff
@@ -7,48 +7,52 @@
...
 Gene expression profiling begins with isolation of total RNA from a tissue sample or from individual cells. mRNA is typically purified from total RNA by removal of ribosomal RNA (rRNA). mRNA is further processed for sequencing, as described in a detailed example protocol found in Appendix 1 of this document.
 
-A visual summary is provided below. [https://rpubs.com/ewilkinson_KRISP/479615](https://rpubs.com/ewilkinson_KRISP/479615)
-**![](https://lh5.googleusercontent.com/lucozpAWSmdTG4KDF7I9Wfua5rvVxX7WlSCdHnaOAS-oaS92o-96hN9OdGyzerGUXUaUIuim4l6JKqeuvAPLUFib_seMquWlu0aL-U4K_WSSIPdhdVUKD4aMXBysAMXoGcGbr4ev)**
+A visual summary is provided below.
+![](https://lh5.googleusercontent.com/lucozpAWSmdTG4KDF7I9Wfua5rvVxX7WlSCdHnaOAS-oaS92o-96hN9OdGyzerGUXUaUIuim4l6JKqeuvAPLUFib_seMquWlu0aL-U4K_WSSIPdhdVUKD4aMXBysAMXoGcGbr4ev)
+
+Figure 2: A basic representation of the major steps and considerations in the sequencing of RNA.
+[(Kukurba et al, 2015)](https://doi.org/10.1101/pdb.top084970)
+
```
Comment (cebriggs7135) [inline context/app/markdown/docs/assay/rnaseq.md:31]:
```suggestion
### *Bulk RNAseq*
For question on bulk RNAseq, contact: [Stephanie Nevins](mailto:snevins@stanford.edu)
```

Comment (cebriggs7135) [inline context/app/markdown/docs/assay/rnaseq.md]:
```suggestion
### *Bulk RNAseq*
For questions on bulk RNAseq, contact: [Stephanie Nevins](mailto:snevins@stanford.edu)
```

Comment (mccalluc) [reply context/app/markdown/docs/assay/rnaseq.md:31]: This is already present below. Doesn't need to be repeated.

**Pull request merged**